<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TodoPage</class>
 <widget class="QWidget" name="TodoPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>489</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTipDuration">
   <number>1</number>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>800</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTipDuration">
        <number>4</number>
       </property>
       <property name="styleSheet">
        <string notr="true">QHeaderView::section
{
	background-color: #f7f7f7;
	font:13px ;
	font-weight: bold;
	border: None;
	line-height: 18px;
	height:48px;

}
 #tableWidget{
	gridline-color: white;
	background-color: rgb(255,255,255);
	selection-color: rgb(0, 0, 0);


}

 #tableWidget::item{
	border-bottom: 1px solid #d6d9dc;
	height:48px;
	
}
#tableWidget::item:selected{
	color: rgb(0, 0, 0);
	background-color: rgb(234, 243, 253);
}


 #tableWidget::item:hover{
	background-color:rgb(234, 243, 253);
}

</string>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::SolidLine</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>48</number>
       </attribute>
       <row>
        <property name="text">
         <string>新建行</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>序号</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>课程号</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>课程名</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>授课教师</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>上课时间</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>上课地点</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>剩余容量</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>总容量</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>操作</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string>211213</string>
        </property>
       </item>
       <item row="0" column="1">
        <property name="text">
         <string>21</string>
        </property>
       </item>
       <item row="0" column="2">
        <property name="text">
         <string>dfadfzdssfasfasfasfadfafd</string>
        </property>
       </item>
       <item row="0" column="3">
        <property name="text">
         <string>adfaf</string>
        </property>
       </item>
       <item row="0" column="4">
        <property name="text">
         <string>gfsg</string>
        </property>
       </item>
       <item row="0" column="5">
        <property name="text">
         <string>sdasfadzf</string>
        </property>
       </item>
       <item row="0" column="6">
        <property name="text">
         <string>sfhgsdg</string>
        </property>
       </item>
       <item row="0" column="7">
        <property name="text">
         <string>sfd</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
